extensions/integration-tests/src/fmkUtils.ts

```diff
@@ -0,0 +1,46 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export enum SuiteType {
+	// Please preserve the capitalized casing and list members in alphabetic order.
+	//
+	Integration = 'Integration',
+	Perf = 'Perf',
+	Stress = 'Stress',
+	Unknown = ''
+}
+
+function toCapitalizedCase(inputString: string): string {
+	return inputString.toLowerCase().replace(/^.|\s\S/g, (a: string) => a.toUpperCase());
+}
+
+let suite: SuiteType = SuiteType.Unknown;
+export function getSuiteType() {
+	if (suite === SuiteType.Unknown) {
+		if (process.env.SuiteType && process.env.SuiteType in SuiteType) {
+			let suiteType: string = toCapitalizedCase(process.env.SuiteType);
+			if (suiteType in SuiteType) {
+				suite = SuiteType[suiteType];
+			}
+		}
+		else {
```

Comment:
In Typescript, generally convention is to have the `else` in the same line as the ending `if`

Reply:
fixed. Good catch, bumped that the code formatter  (format document in vscode) does not fix it.

Reply:
There are a few other places I noticed this actually.